frontend/src/components/QueueTable.js

```diff
@@ -98,20 +106,17 @@ function QueueTable({ queueData: { name, rows } }) {
             render: ({ id }) => <NotesButton id={id} />,
           },
         ]}
-      />
-      <NotesDialog
-        visibleDialog={visibleDialog}
-        toggleVisibleDialog={toggleVisibleDialog}
```

Comment:
Is `NotesDialog` supposed to be added back in somewhere?

Reply:
I'm not sure how it got cut.  You made it, didn't you? Is this where it should be?

Reply:
Putting it back in

Reply:
Fixed. Sorry for cutting it